nessie.go

```diff
@@ -922,7 +937,36 @@ func (n *nessusImpl) Permissions(objectType string, objectID int64) ([]Permissio
 	return reply, nil
 }
 
-// DeletePolicy Delete a policy
+// CreatePolicy Create a policy.
+func (n *nessusImpl) CreatePolicy(createFolderRequest CreatePolicyRequest) (CreatePolicyResp, error) {
+	if n.verbose {
+		log.Println("Creating a policy...")
+	}
+
+	resp, err := n.doRequest("POST", "/policies", createFolderRequest, []int{http.StatusOK})
+	if err != nil {
+		return CreatePolicyResp{}, err
+	}
+
+	defer resp.Body.Close()
+	var reply CreatePolicyResp
+	if err = json.NewDecoder(resp.Body).Decode(&reply); err != nil {
+		return CreatePolicyResp{}, err
+	}
+	return reply, nil
+}
+
+// ConfigurePolicy Changes the parameters of a policy.
+func (n *nessusImpl) ConfigurePolicy(policyID int64, createFolderRequest CreatePolicyRequest) error {
```

Comment:
Can you change `createFolderRequest` into simply `req`? Seems like bad copy&paste

Reply:
Nice catch, Certainly!